files/build_templates/docker_image_ctl.j2

```diff
@@ -540,7 +540,9 @@ start() {
     # TODO: Mellanox will remove the --tmpfs exception after SDK socket path changed in new SDK version
 {%- endif %}
     docker create {{docker_image_run_opt}} \
+{%- if '--net' not in docker_image_run_opt %}
         --net=$NET \
```

Comment:
I see the value fo ```$NET``` is ```host``` when the container is multi ASIC agnostic. So, I assume you may want to block having ```asic-service``` set to ```True``` and specifying custom ```network``` in the manifest. Could you please add such check in sonic-utilities PR?